evap/evaluation/templates/navbar.html

```diff
@@ -73,6 +73,7 @@
                             <a class="dropdown-item" href="{% url 'staff:degree_index' %}">{% trans 'Degrees' %}</a>
                             <a class="dropdown-item" href="{% url 'staff:index' %}">{% trans 'Email templates' %}</a>
                             <a class="dropdown-item" href="{% url 'staff:faq_index' %}">{% trans 'FAQ' %}</a>
+                            <a class="dropdown-item" href="{% url 'staff:infotexts' %}">{% trans 'Infotexts' %}</a>
```

Comment:
Maybe "Edit infotexts"?

Reply:
Almost all of these pages are meant for editing, so I would keep it short. "Infotexts" is fine :)